edx_name_affirmation/tests/test_signals.py

```diff
@@ -0,0 +1,58 @@
+"""
+Tests for Name Affirmation signals
+"""
+
+import ddt
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from edx_name_affirmation.signals import _listen_for_lms_retire_verified_names
+
+from edx_name_affirmation.models import VerifiedName
+
+User = get_user_model()
+
+
+@ddt.ddt
+class RetirementSignalVerifiedNamesTest(TestCase):
+    """
+    Tests for the LMS User Retirement signal for Verified Names
+    """
+
+    def setUp(self):
```

Comment:
Does this setup for test cases look good?

Reply:
I think so!

If you wanted to make it look a little tidier, you could consider implementing several factories that would take care of some of the boiler plate of setting the `name`, `profile_name`, etc. There are some good examples in [edx-exams](https://github.com/edx/edx-exams/blob/main/edx_exams/apps/core/test_utils/factories.py), but I also think it would be fine to leave as is because the test suite is relatively small!

Reply:
I had this thought but then knowing me, it would drive me mad to not to then update all of the tests with these factories... so if it's alright, I may just leave as is in the interest of getting this out quicker (especially giving all the local testing woes for this one)? But thank you for the feedback!